src/Tasks/Resources/Strings.resx

```diff
@@ -2985,6 +2985,18 @@
   <data name="ResolveKeySource.PfxUnsupported" xml:space="preserve">
     <value>PFX signing not supported on .NET Core.</value>
   </data>
+
+  <!--
+        MSB3991 - MSB3999   Task: CombineTargetFrameworkInfoProperties
+  -->
+  <data name="CombineTargetFrameworkInfoProperties.NotNullAndEmptyRootElementName">
+    <value>MSB3991: '{0}' is not set or empty. When UseAttributeForTargetFrameworkInfoPropertyNames is false, make sure to set a non-empty value for '{0}'.</value>
```

Comment:
Does this need to be parameterized? It already has one attribute name hardcoded.

Reply:
Parametrize all of them